### PR TITLE
janitoring: Fixed grammar.

### PR DESCRIPTION
**Pull request by JakubNvk**
Signed-off-by: Jakub Novak <jakubn@imterra.com>
Reviewed-by: Peter Polacik <pepol@imterra.com>

### CODE REVIEW

File: workflow.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Development Workflow
 
-This document should describe basic workflow of developer on Imterra projects.
+This document should describe basic workflow of a developer on Imterra projects.
```
Comment (pepol) [inline workflow.md]: This line is 81 characters long. I think we should follow the 80-character limit in documents as well. What's your opinion?

File: workflow.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ work in a clone of upstream.
 #### Branch naming
 
 Recommended branch naming is `username/featurename`, where `username` is
-either *priv*/*foss* username of creator, or other generally accepted and
-well known identifier of creator. Please, make sure that this identifier is
-unique. Also please **be consistent** with your naming. That means, you
+either *priv*/*foss* username, or other generally accepted and well known 
+identifier of the creator. Please, make sure that this identifier is unique.
```
Comment (pepol) [inline workflow.md]: Please remove the spaces at the end of lines 28 and 29.

**Comment by pepol** [comment]
@JakubNvk Thanks a lot for those grammar fixes. Please take a look at my comments, otherwise LGTM!

File: workflow.md (hunk 1 above)
Comment (JakubNvk) [inline workflow.md]: @pepol youre right, lets not make any exceptions!

**Comment by JakubNvk** [comment]
@pepol ping

**Comment by pepol** [comment]
@JakubNvk one more comment, thanks!

**Comment by pepol** [comment]
@JakubNvk thanks, LGTM, please squash them so we can merge this.

**Comment by pepol** [comment]
LGTM

**Pull request merged**